### PR TITLE
Bump quarkus dependencies

### PR DESCRIPTION
**Pull request by sixdouglas**
Let's refresh the dependencies.

The `solution` branch will be rebased after merging this one.

### CODE REVIEW

File: order-api/pom.xml [hunk 1]
```diff
@@ -6,17 +6,15 @@
...
-    <surefire-plugin.version>2.22.1</surefire-plugin.version>
-    <maven.compiler.target>11</maven.compiler.target>
-    <quarkus.platform.version>1.9.2.Final</quarkus.platform.version>
-    <maven.compiler.source>11</maven.compiler.source>
+    <compiler-plugin.version>3.11.0</compiler-plugin.version>
+    <maven.compiler.release>17</maven.compiler.release>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.parameters>true</maven.compiler.parameters>
```
Comment (loicmathieu) [inline order-api/pom.xml:16]: I think you should keep `<maven.compiler.parameters>true</maven.compiler.parameters>`

Comment (loicmathieu) [reply order-api/pom.xml:16]: Same for other pom.xml file

File: order-enhancer/pom.xml [hunk 2]
```diff
@@ -41,7 +39,7 @@
     <dependency>
       <groupId>io.smallrye.reactive</groupId>
       <artifactId>smallrye-reactive-messaging-in-memory</artifactId>
-      <version>2.7.0</version>
+      <version>4.9.0</version>
```
Comment (loicmathieu) [inline order-enhancer/pom.xml]: It should be in the Quarkus BOM now, can you check and remove the version if it's managed there?

**Review by loicmathieu — CHANGES_REQUESTED** [review]
Thanks for doing this, some minor remarks

File: order-api/pom.xml (hunk 1 above)
Comment (sixdouglas) [reply order-api/pom.xml:16]: ok put it back

File: order-enhancer/pom.xml (hunk 2 above)
Comment (sixdouglas) [reply order-enhancer/pom.xml]: in deed, it is better ths way

File: order-enhancer/pom.xml [hunk 3]
```diff
@@ -6,17 +6,15 @@
...
-    <surefire-plugin.version>2.22.1</surefire-plugin.version>
-    <maven.compiler.target>11</maven.compiler.target>
-    <quarkus.platform.version>1.9.2.Final</quarkus.platform.version>
-    <maven.compiler.source>11</maven.compiler.source>
+    <compiler-plugin.version>3.11.0</compiler-plugin.version>
+    <maven.compiler.release>17</maven.compiler.release>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.parameters>true</maven.compiler.parameters>
```
Comment (loicmathieu) [inline order-enhancer/pom.xml:16]: Please add back compiler.parameters = true

**Review by loicmathieu — CHANGES_REQUESTED** [review]

File: order-enhancer/pom.xml (hunk 3 above)
Comment (sixdouglas) [reply order-enhancer/pom.xml:16]: done

Comment (loicmathieu) [reply order-enhancer/pom.xml:16]: Thanks

**Pull request merged**